grisera_api/tests/tests_mongodb/test_observable_information.py

```diff
@@ -0,0 +1,67 @@
+import mongomock
+
+from observable_information.observable_information_model import ObservableInformationIn
+from registered_channel.registered_channel_model import RegisteredChannelIn
+from tests.tests_mongodb.utils import MongoTestCase
+from models.not_found_model import NotFoundByIdModel
+from services import Services
+from registered_data.registered_data_model import RegisteredDataIn
+from mongo_service.mongodb_api_config import mongo_api_host, mongo_api_port
+from recording.recording_model import RecordingIn
+
+
+class TestMongoRegisteredData(MongoTestCase):
+    def generate_recording(self, save: bool):
+        recording = RecordingIn()
+        if not save:
+            return recording
+        service = Services().recording_service()
+        return service.save_recording(recording)
+
+    @mongomock.patch(servers=((mongo_api_host, mongo_api_port),))
+    def test_create(self):
+        service = Services().observable_information_service()
+        recording = self.generate_recording(save=True)
+        observable_infromation = ObservableInformationIn(recording_id=recording.id)
```

Comment:
please replace all `infromation` to `information` in this file

Reply:
fixed